doc/dev/developer_guide/index.rst

```diff
@@ -1628,12 +1628,22 @@ it would execute a single test.
           compatible with ``python2``. Therefore, use ``python2`` to run the
           tests locally.
 
-vstart_runner.py can take 3 options -
+vstart_runner.py can take 4 options -
```

Comment:
nit, s/4/following/ so we don't need to keep counting them when adding more options in future.